pilot/cmd/pilot-agent/status/server_test.go

```diff
@@ -662,14 +662,6 @@ func TestAppProbe(t *testing.T) {
 			podIP:      "::1",
 			ipv6:       true,
 		},
-		{
-			name:       "tcp-livez-wrapped-ipv6",
-			probePath:  "app-health/hello-world/livez",
-			config:     simpleTCPConfig,
-			statusCode: http.StatusOK,
-			podIP:      "[::1]",
```

Comment:
its a valid ip, still useful to test IMO

Reply:
it's okay to leave it and I will change the expected test result